.github/CODEOWNERS

```diff
@@ -2,6 +2,8 @@
 
 ## Changes to .github
 .github/* @mustermeiszer @NunoAlexandre @lemunozm
+.github/workflows @gpmayorga
+.github/actions @gpmayorga
```

Comment:
👌🏻

Reply:
we could add @wischli too, for the bus factor